spec/requests/api/journeys_controller_update_spec.rb

```diff
@@ -78,6 +78,11 @@
           expect { do_patch }.to change(GenericEvent::JourneyUpdate, :count).by(1)
         end
 
+        it 'creates a JourneyUpdate generic event' do
```

Comment:
could you update the description of the test

Reply:
:facepalm: Done!